lldap/lldap
# Rootless container

Testing build for rootless container
1. Add rootless entrypoint 
2. Add Dockerfile for rootless
3. Add rootless meta
4. Build rootless images

Run as user 1000 (a.k.a lldap) by default. Deployer will be responsible for permission setup, only check `is config file exist?`.

#755 

Things to do
- [x] Temp base image. Local service registry route.
- [x] Rename docker-entry-point for rootless.
- [x] Add comments push behaviour on dummy registry
- [x] Gosu fetch from github and only on root image

Some resources:
* https://docs.docker.com/build/ci/github-actions/share-image-jobs/
* https://docs.docker.com/build/ci/github-actions/local-registry/
